test/verify/check-docker

```diff
@@ -555,15 +555,16 @@ CMD ["/bin/sh"]
 
         m.execute("systemctl start docker || systemctl start docker-latest")
 
-        # start abrt-ccpp service until abrt-journal-core patch is released
-        # https://github.com/abrt/abrt/commit/da2ae52f307f05f2eac0ad6e647561633544722a
-        m.execute("systemctl start abrt-ccpp")
+        m.execute("systemctl start abrt-journal-core")
```

Comment:
this line is not needed, works even without it

Reply:
Nice! I'll drop it, most tests didn't run yet anyway. Queue went crazy again, PR #220 had ~ 550 queue entries (will investigate next week). For now I just restarted it.